dissect/target/helpers/configutil.py

```diff
@@ -254,6 +256,81 @@ def parse_file(self, fh: TextIO) -> None:
         self.parsed_data = {"content": fh.read(), "size": str(fh.tell())}
 
 
+class Xml(ConfigurationParser):
+    """Parses an XML file. Ignores any constructor parameters passed from ``ConfigurationParser`."""
+
+    def _tree(self, tree: ElementTree) -> dict:
```

Comment:
Im suggesting the following, since I find the `tag`key a little redundant. 

With this change, and the changes on lines 277, 297 and 321, the `tag` key is removed from the dictionary for a single `root` key. This `root` key which then holds all the `nodes` (elements) as per the current implementation. 

This way you still have all the information you previously had, and will be playing nicer with the file system-esque nature of the `config_tree` plugin. 
```suggestion
    def _tree(self, tree: ElementTree, root: bool = False) -> dict:
```